src/modules/common/Config.ts

```diff
@@ -53,6 +58,7 @@ export class Config implements IConfig {
         this.database = new DatabaseConfig();
         this.logging = new LoggingConfig();
         this.consensus = new ConsensusConfig();
+        this.votera = new VoteraConfig();
```

Comment:
Votera config must be an option.
It may be driven by the configuration of the votera, or there may be stoa that does not require a votera.
You need to add to the config whether to request the votera server meta information.

Reply:
done